themes/gatsby-theme-deriv/src/common/websocket/index.ts

```diff
@@ -41,7 +41,12 @@ export class ApiManager {
     public init(lang?: string) {
         if (!this.ready) {
             if (!this.socket) {
-                const language = lang === 'ach' ? getCrowdin() : lang?.replace('-', '_')
+
+                const pathname = isBrowser() ? window.location.pathname : '';
+                const isAcademySignup = pathname.includes('/academy-signup');
+
+                const language = isAcademySignup ? 'EN' : (lang === 'ach' ? getCrowdin() : lang?.replace('-', '_'));
```

Comment:
app_id: 16929
l: en
brand: deriv

language is simple en
```suggestion
                const language = isAcademySignup ? 'en' : (lang === 'ach' ? getCrowdin() : lang?.replace('-', '_'));
```